<launch>  
  <node pkg="steerforce_conv" type="steerforce_converter_sim.py" name="steerforce_converter" output="screen">
    <rosparam file="$(find steerforce_conv)/params/params_sim.yaml" />
  </node>
</launch>